Corzbank.Web/ClientApp/src/app/header/header.component.ts

```diff
@@ -22,4 +28,51 @@ export class HeaderComponent implements OnInit {
       console.log('The dialog was closed');
     });
   }
+
+  get isAuthenticated() {
+    let isLogedin = Number(this.storageService.getItem(StorageTypeEnum.localStorage, "isLogedin"));
```

Comment:
I capital.
Double g.

Reply:
To validate a boolean from string do not use Boolean(), because Boolean("false") = true
Use string === "true".

Reply:
resolved